spec.html

```diff
@@ -30604,12 +30604,12 @@ <h1>
       <emu-clause id="sec-utc-t" type="abstract operation">
         <h1>
           UTC (
-            _t_: unknown,
+            _t_: a Number,
           )
         </h1>
         <dl class="header">
           <dt>description</dt>
-          <dd>It converts _t_ from local time to UTC.</dd>
+          <dd>It converts _t_ from local time to a UTC time value.</dd>
```

Comment:
I would prefer this wording to be more similar to the wording above:

> It converts time value _t_ from UTC to local time.

Reply:
What does "from local time" add here?

Reply:
It communicates the semantics of the value, albeit poorly (as noted in the PR description, a bigger refactoring of the Date AOs is still in order). But specifically, _t_ is a Number representing a local date and time of day equal to the UTC date and time of day that correspond with (mis)interpreting it as a time value and ignoring the bounds that normally apply.